.babelrc

```diff
@@ -19,10 +19,12 @@
                     "@": "./",
                     "@server": "./server",
                     "@client": "./client",
+                    "@components": "./client/pages/Home/Components",
                     "@pages": "./client/pages",
                     "@utils": "./server/utils",
                     "@models": "./server/models",
                     "@config": "./server/config",
+                    "@images": "./client/images",
```

Comment:
Why do we have this? Images shouldn't be stored on the client. They should be stored on the server as express static files.

Reply:
Apologies. I forgot to remove that after uninstalling file-loader